e2e-test/src/test/java/io/cdap/plugin/file/stepsdesign/FileConnector.java

```diff
@@ -0,0 +1,379 @@
+    out.close();
+  }
+
+  @Then("Validate successMessage is displayed")
+  public void validateSuccessMessageIsDisplayed() {
+    CdfLogActions.validateSucceeded();
+  }
+
+  @Then("Get Count of no of records transferred to BigQuery in {string}")
+  public void getCountOfNoOfRecordsTransferredToBigQueryIn(String tableName) throws IOException, InterruptedException {
+    int countRecords = GcpClient.countBqQuery(E2ETestUtils.pluginProp(tableName));
+    BeforeActions.scenario.write("**********No of Records Transferred******************:" + countRecords);
+    Assert.assertTrue(countRecords > 0);
+  }
+
+  @Then("Delete the table {string}")
+  public void deleteTheTable(String tableName) throws IOException, InterruptedException {
+    GcpClient.dropBqQuery(E2ETestUtils.pluginProp(tableName));
+    BeforeActions.scenario.write("Table Deleted Successfully");
+  }
+
+  @Then("Verify output field {string} in target BigQuery table {string} contains source file path {string}")
+  public void verifyOutputFieldInTargetBigQueryTableContainsSourceFilePath(
+    String outputField, String targetTable, String filePath) throws IOException, InterruptedException {
+    Optional<String> result = GcpClient
+      .getSoleQueryResult("SELECT distinct " + E2ETestUtils.pluginProp(outputField) + " as bucket FROM `"
+                            + (E2ETestUtils.pluginProp("projectId")) + "."
+                            + (E2ETestUtils.pluginProp("dataset")) + "."
+                            + E2ETestUtils.pluginProp(targetTable) + "` ");
+    String pathFromBQTable = StringUtils.EMPTY;
+    if (result.isPresent()) {
+      pathFromBQTable = result.get();
+    }
+    BeforeActions.scenario.write("GCC bucket path in BQ Table :" + pathFromBQTable);
+    Assert.assertEquals("file:" + E2ETestUtils.pluginProp(filePath), pathFromBQTable);
+  }
+
+  @Then("Verify datatype of field {string} is overridden to data type {string} in target BigQuery table {string}")
+  public void verifyDatatypeOfFieldIsOverriddenToDataTypeInTargetBigQueryTable(
+    String field, String dataType, String targetTable) throws IOException, InterruptedException {
+    Optional<String> result = GcpClient
+      .getSoleQueryResult("SELECT data_type FROM `" + (E2ETestUtils.pluginProp("projectId")) + "."
+                            + (E2ETestUtils.pluginProp("dataset")) + ".INFORMATION_SCHEMA.COLUMNS` " +
+                            "WHERE table_name = '" + E2ETestUtils.pluginProp(targetTable)
+                            + "' and column_name = '" + E2ETestUtils.pluginProp(field) + "' ");
+    String dataTypeInTargetTable = StringUtils.EMPTY;
+    if (result.isPresent()) {
+      dataTypeInTargetTable = result.get();
+    }
+    BeforeActions.scenario.write("Data type in target BQ Table :" + dataTypeInTargetTable);
+    Assert.assertEquals(E2ETestUtils.pluginProp(dataType),
+                        dataTypeInTargetTable.replace("64", StringUtils.EMPTY).toLowerCase());
```

Comment:
why are you doing this?

Reply:
In BQ INFORMATION_SCHEMA table data_type is "FLOAT64" and on CDAP UI it is getting displayed as only "float".